.circleci/config.yml

```diff
@@ -17,6 +17,25 @@ executors:
     working_directory: ~/repo/pepper-apis
 
 commands:
+  install-tcell:
+    steps:
+      - run:
+          name: Download tcell files from bucket
+          command: |          
+            TAR_FILE_URL=`gsutil ls $TCELL_PATH  | sort -r | head -1`
+
+            if [ -z $TAR_FILE_URL ]
+            then
+              echo "Could not find tcell archive ${TCELL_PATH}"
+              exit 1
+            fi
+            TAR_FILE_PATH="/tmp/$(basename -- $TAR_FILE_URL)"
+            gsutil -u broad-ddp-prod cp  $TAR_FILE_URL $TAR_FILE_PATH
+            cd ~/repo/ddp-workspace
```

Comment:
Going to have to run this, but maybe this is meant for ddp-angular? I don't think we have a ddp-workspace directory in `ddp-study-server` repo.

Reply:
This path works?

Reply:
Oops, let me  clean this up.